Mods/SML/Source/SML/Public/Toolkit/KismetBytecodeDisassembler.h

```diff
@@ -5,21 +5,32 @@
 
 class SML_API FSMLKismetBytecodeDisassembler {
 public:
-	/** Converts a single expression into json object */
+	/** Converts a single expression into json object, provided Script and SelfScope have been set */
 	TSharedPtr<FJsonObject> SerializeExpression(int32& ScriptIndex);
 
 	/** Parses a block of statements until it hits return */
 	TArray<TSharedPtr<FJsonValue>> SerializeFunction(UStruct* Function);
 
+	/** Converts a single statement at the given index into json object. Will throw an error if the statement index does not correspond to the start of a statement.*/
+	TSharedPtr<FJsonObject> SerializeStatement(UStruct* Function, int32 StatementIndex);
+
```

Comment:
After using `GetStatementLength` in `InstallBlueprintHook`, this function will be unused.

Reply:
Removed